Makefile.sp

```diff
@@ -114,7 +115,24 @@ sp-vale: sp-install
 	@echo ""
 	@. $(VENV); vale --config "$(SPHINXDIR)/vale.ini" --glob='*.{md,txt,rst}' $(TARGET)
 
-
+sp-pdf-prep: sp-install
+	@for packageName in $(REQPDFPACKS); do (dpkg-query -W -f='$${Status}' $$packageName 2>/dev/null | \
+        grep -c "ok installed" >/dev/null && echo "Package $$packageName is installed") && continue || \
+        (echo "\nPDF generation requires the installation of the following packages: $(REQPDFPACKS)" && \
+        echo "" && echo "Run sudo make pdf-prep-force to install these packages" && echo "" && echo \
+        "Please be aware these packages will be installed to your system") && break ; done
```

Comment:
I just uninstalled one of the packages and tried again, and it gave the expected message but then continued with the build - can we make sure the target fails?
Otherwise users will get confused, because the error at the end doesn't indicate the missing package.

Reply:
I now realise I replaced `false` with `break` instead of `exit 1`. I really should have checked the full workflow rather than pieces. Should be sorted now.